tests/conftest.py

```diff
@@ -65,18 +65,19 @@ def pytest_report_header(config):
     return [f"asyncio loop: {loop}"]
 
 
-def pytest_sessionstart(session):
-    # Configure the async loop.
-    loop = session.config.getoption("--loop")
-    if loop == "uvloop":
-        import uvloop
-
-        uvloop.install()
-    else:
-        assert loop == "default"
-
-    if sys.platform == "win32":
-        asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
+asyncio_options = {}  # type: ignore[var-annotated]
```

Comment:
This should do too, with the right imports:

```suggestion
asyncio_options: Dict[str, Any] = {}
```

Reply:
Uhm... but mostly, why do we need a global `asyncio_options` module-level dictionary at all here and don't we do everything in the fixture itself?

Reply:
The global `asyncio_options` is imported from elsewhere (from `tests/pool/conftest.py` in this PR, and from `test_waiting.py`, at least temporarily in   https://github.com/psycopg/psycopg/pull/146/commits/89bdf08bfe660a1a345d582583fc7b01606e1f01#diff-ec162367f2b494baea5f25f3d81908ca9edbbae2b6f129fd2b3bb3c89d626a20).